ui/pages/confirm-approve/confirm-approve-content/index.scss

```diff
@@ -27,6 +27,18 @@
     position: absolute;
   }
 
+  &__metafoxlogo {
+    .app-header {
+      &__metafox-logo--horizontal {
+        display: none;
+      }
+
+      &__metafox-logo--icon {
+        display: block;
```

Comment:
This is needed as an override since this is set to hide on full screen views here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/onboarding-flow/onboarding-app-header/index.scss#L25